application/src/main/java/org/opentripplanner/updater/siri/AddedTripBuilder.java

```diff
@@ -81,11 +82,15 @@ class AddedTripBuilder {
     Objects.requireNonNull(newServiceJourneyRef, "EstimatedVehicleJourneyCode is required");
     tripId = entityResolver.resolveId(newServiceJourneyRef);
 
-    //OperatorRef of added trip
+    // OperatorRef of added trip
     Objects.requireNonNull(estimatedVehicleJourney.getOperatorRef(), "OperatorRef is required");
     String operatorRef = estimatedVehicleJourney.getOperatorRef().getValue();
     operator = entityResolver.resolveOperator(operatorRef);
 
+    // DataSource of added trip
+    Objects.requireNonNull(estimatedVehicleJourney.getDataSource(), "DataSource is required");
```

Comment:
Do you really have to have a data source? Isn't that going to break Skanetrafiken's updates?

Reply:
@habrahamsson-skanetrafiken - Do you not set datasource? I thought you said you use this in today's meeting, though you just have one.

Reply:
Making it optional.